accepted/2020/net5/net5.md

```diff
@@ -502,6 +502,53 @@ NuGet uses the long form `.NETFramework,Version=4.5` in the dependency groups.
 We may want to change NuGet to allow the friendly name there as well and update
 our packaging tools to re-write to friendly name on pack._
 
+### Mapping to long names
+
+As [mentioned earlier](#mapping-to-properties), TFMs have multiple formats. The
+most common ones are the friendly name (`netstandard2.0`) and the long name
+(`.NETStandard, Version=2.0`). NuGet also uses some other encodings (such as
+`.NETStandard2.0`).
+
+All places that have to refer to a NuGet asset need a naming encoding that can
+represent the new TFMs which now also includes platform information. We'd like
+to avoid using a long name like
+
+```text
+.NETCoreApp, Version=5.0, Platform=iOS, PlatformVersion=13.0
+```
+
+The reason being that the NuGet short name is essentially used to encode both a
+target framework and a target platform, both of which already have a canonical
+long name form.
+
+Since the friendly name has become the primary name that developers author in
+the developer experience (project file, NuGet folder names, etc) we'd like to
+standardize on that encoding. Unfortunately, NuGet itself isn't consistent with
+using the friendly name today. We generally don't want to break backwards
+compatibility, that is existing frameworks should be encoded in the way they are
+encoded today, be that friendly name, long name or some custom encoding.
+
+However, for frameworks in the .NET 5 era (and later) we want to use
```

Comment:
Probably should be, "we want to consistent use".

Reply:
Good call. Fixed.